apps/desktop-wallet/src/api/addressesTokensPricesDataHooks.ts

```diff
@@ -102,21 +105,71 @@ export const useSortTokensByWorth = (tokens: Asset[]) => {
   )
 }
 
+type TokenId = string
+
 export const useAddressesTokensWorth = (addressHash?: AddressHash) => {
```

Comment:
I changed the implementation of this hook to return a `Record` that maps a tokenID to the total worth. Later on I introduced the `useAddressesTokensTotalWorth` that implements the previous behavior of `useAddressesTokensWorth`.